public/css/main.css

```diff
@@ -1,29 +1,69 @@
 a, a:hover, a:active, a:visited, a:focus {
     text-decoration: none;
 }
-.calendar:hover {
-  color: #6495ED;
-}
-.calendar {
-  color: gray;
+body {
+  background-color: #F0EAEA;
```

Comment:
These colours are still off - sorry I thought I'd sent them on Gitter but apparently not. I'll add them to the issue

Reply:
:+1: